index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <script type="text/javascript" src="dist/envspec.js"></script>
+     <link rel="stylesheet" href="dist/styles.css">
+  </head>
+
+  <body>
+    <script type="text/javascript" src="dist/demoPage.js"></script>
+    <p><h1>.ENV.SPEC demo page</h1></p>
+    <p><h4><i>Type or paste your .env.spec file in the text area below :</i></h4></p>
+
+    <div>
+        <textarea id="inputArea" placeholder="Start here..." rows = "6" cols = "30"></textarea>
+        <button value="Get HTML" onclick="getHTML()"> Get HTML</button>
+    </div>
+    <div>
+        <textarea id="outputArea" placeholder="HTML text" rows = "8" cols = "60" readonly></textarea>
+    </div>
+
+    <p><h4><i>HTML RESULT :</i></h4></p>
```

Comment:
1. Remove the parent `<p>` tag, as `<h4>` tags cannot be put inside paragraphs.
2. Remove the `<i>` tag. We will apply styling later with CSS
3. Rename `HTML RESULT :` to `Rendered form:`